tests/unit/utils/test_sqlalchemy_query_utils.py

```diff
@@ -201,10 +201,7 @@ def test_generated_table_from_query():
     )
     table = GeneratedTable.from_query("some_table", query, schema="some_schema")
     assert str(sqlalchemy.schema.CreateTable(table)).strip() == (
-        "CREATE TABLE some_schema.some_table (\n"
-        "\tnumber INTEGER, \n"
-        "\tstring VARCHAR\n"
-        ")"
+        "CREATE TABLE some_schema.some_table (\n\tnumber INTEGER, \n\tstring VARCHAR\n)"
```

Comment:
And `textwrap.dedent` again here?

Reply:
I've changed the others you pointed out (and a couple more), but I've left this one because with dedent it ends up with trailing spaces at the end of the lines, which `just fix` removes, and breaks the test

Reply:
Uggh, yeah. Thanks/sorry!